sites/public/src/lib/helpers.tsx

```diff
@@ -145,6 +145,16 @@ export const getListings = (listings: Listing[]) => {
   })
 }
 
+export const getZeroListingsCard = (listings: Listing[]) => {
```

Comment:
I know that getListings lives here but this is small enough to just inline into in the listingslist component IMO

Reply:
Done.